bentoml/_internal/cli/bento_management.py

```diff
@@ -15,27 +17,26 @@
 
 from ..utils import calc_dir_size
 from ..utils import human_readable_size
-from .click_utils import _is_valid_bento_tag
-from .click_utils import _is_valid_bento_name
+from .click_utils import is_valid_bento_tag
+from .click_utils import is_valid_bento_name
 from ..yatai_client import yatai_client
 from ..configuration.containers import BentoMLContainer
 
 if TYPE_CHECKING:
     from ..bento import BentoStore
-
 logger = logging.getLogger(__name__)
 
 
 def parse_delete_targets_argument_callback(
-    ctx, params, value
-):  # pylint: disable=unused-argument
+    ctx: "click.Context", params: "click.Parameter", value: t.Any
+) -> t.Any:  # pylint: disable=unused-argument
     if value is None:
         return value
     delete_targets = value.split(",")
     delete_targets = list(map(str.strip, delete_targets))
     for delete_target in delete_targets:
         if not (
-            _is_valid_bento_tag(delete_target) or _is_valid_bento_name(delete_target)
+            is_valid_bento_tag(delete_target) or is_valid_bento_name(delete_target)
```

Comment:
We should probably introduce `Tag.is_valid` or just use `Tag(<tag to check>)` here.

Reply:
yes we should, will leave that to follow up pr